src/test/java/com/wcc/platform/factories/SetupFactories.java

```diff
@@ -217,4 +245,14 @@ public static PageSection createPageSectionTest(final String title) {
         createLinkTest(),
         List.of("topic1 " + title, "topic2 " + title));
   }
+
+  /**
+   * Factory test for repository file section to delete the content of the file used for testing.
+   */
+  public static void deleteFile(final File file) {
+    if (file.exists() && !file.delete()) {
+      final String msg = "Error deleting file " + file.getAbsolutePath();
+      throw new RuntimeException(msg);
```

Comment:
This is a factory for test, I don't think we need any exception or log. Because if does not exist it is ok, we want to remove if exist.